src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/FalconElbow/FalconElbowConstants.java

```diff
@@ -10,6 +12,18 @@ public class FalconElbowConstants {
 
     public static final int MOTOR_ID = 1;
 
+    public static final int HARD_REVERSE_LIMIT_SWITCH_ID = 1;
+
+    public static final int HARD_FORWARD_LIMIT_SWITCH_ID = 1;
+
+    public static final double kS = 1;
+
+    public static final double kV = 1;
+
+    public static final double kG = 1;
+
+    public static final double kA = 1;
+
     public static final SimpleMotorFeedforward SIMPLE_MOTOR_FEED_FORWARD = new SimpleMotorFeedforward(kS, kV, kA);
```

Comment:
I know it is not in this PR but why don't we use the `ArmFeedForward` object from wpilib?

Reply:
in another cr about the motion , same about al things related

Reply:
but you did change it in `src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/NeoElbow/NeoElbowConstants.java`

Reply:
right but all the motion planning pid ff mm will be in another cr and we dont want deep in it in this cr

Reply:
yoav - never mind did it